update(UpdateTool): present solution when translation are missing from Bonita App Directory (#2078)
bonitasoft/bonita-doc: modules/version-update/pages/update-with-update-tool.adoc

For backward compatibility checks, refer to the xref:ROOT:release-notes.adoc[release notes].

// link to bonita client library => to decomment

> In addition, if your application connects to Bonita Engine using the HTTP access mode, see the xref:ROOT:configure-client-of-bonita-bpm-engine.adoc[bonita-client library] documentation page.

Reviewer: ```suggestion
In addition, if your application connects to Bonita Engine using the HTTP access mode, see the xref:api:configure-client-of-bonita-bpm-engine.adoc[bonita-client library] documentation page.
```

== Troubleshooting

> === Translation are missing in application directory app

Reviewer: it is not really the app that has missing translation but the pages 
```suggestion
=== Translation are missing in Application Directory and Admin Case list
```

> _Symptom:_ Starting from Bonita 2021.2, when creating an application using pages available in ressources list of previous Bonita versions, translations may be missing in Bonita Application Directory. 
> _Cause:_ The translations are in localization.json and not in the exported pages. 
> _Solution:_ If an app has been created using a page exported from the resources list in a certain version, you will need to either reexport the pages from the new version or update your app by adding the localization.json of the new version. This way the right translations will be available. 

Reviewer: ```suggestion
_Symptom:_ Application Directory and Admin Case List pages are only displayed in English. 
_Cause:_ The translations are missing in the file localization.json in the pages zip content. 
_Solution:_ If a page has been created using, as template, Bonita Application Directory or Admin Case List exported from the resources list in a version older than 2022.2 (2022.1-u1 for subscription editions), you will need to either reexport the pages from the new version or update your app by adding the localization.json of the new version. This way the right translations will be available. 
```
